src/api-next/api-next.ts

```diff
@@ -1995,7 +2016,10 @@ export class IBApiNext {
           undefined,
           [[EventName.nextValidId, this.onNextValidId]]
         )
-        .pipe(map((v: { all: number }) => v.all))
+        .pipe(map((v: { all: number }) => v.all)),
+      {
+        defaultValue: -1,
```

Comment:
same here

Reply:
Same as above, we would need to change function signature to indicate that it can return undefined, will actually will never do it because onNextValidId emits and completes all at once.